helpers/kubernetes/cluster.go

```diff
@@ -208,6 +197,44 @@ func (c *Cluster) IsPodRunning(ctx context.Context, podName, namespace string) w
 	}
 }
 
+// IsPodDone returns a condition function that indicates whether the given pod has
+// terminated in some way or not. Use this for oneshot pods, i.e. pods launched by a Job
+// and expected to terminate.  The pod is considered terminated if __at least one__ of its
+// container is terminated. This condition takes into account that even a one-shot job may
+// contain a non-terminating container (linkerd!)
+
```

Comment:
This newline breaks the doc display in my editor.

Reply:
Will fix.

Reply:
Done. Commit 7eb20b7e67e5b4cff825d27bb2309b3486e1cc68.